packages/jest-haste-map/src/crawlers/node.js

```diff
@@ -124,6 +124,7 @@ module.exports = function nodeCrawl(
   extensions: Array<string>,
   ignore: IgnoreMatcher,
   data: InternalHasteMap,
+  forceNativeFind: boolean = false,
```

Comment:
can we make this an object instead please? That way we can add more options later.